docs/sources/panels-visualizations/visualizations/table/index.md

```diff
@@ -119,60 +129,73 @@ value4 , value5 , value6
 
 If you need to hide columns, you can do so using [data transformations](ref:data-transformation), [field overrides](#field-overrides), or by [building a query](ref:build-query) that returns only the needed columns.
 
-If you’re using a cell type such as sparkline or JSON, the data requirements may differ in a way that’s specific to that type. For more info refer to [Cell type](#cell-type).
-
 ## Debugging in tables
```

Comment:
I feel like this section is from another story.

Reply:
Do you mean you don't think it fits here? If so, is there somewhere else you'd suggest putting it? We wanted to highlight that this is a thing you can do, so we wanted to make it prominent on this page.

Reply:
I'm not sure where's a good place for it - but to me this is not related to table the visualization as you don't have any way to access options, customize it. I also feel like debugging is a big word for this. You're just visualizing your data in a table. But also it's been like this for a while and nobody said anything..  😅

Reply:
Okay, I'm gonna take it out. Maybe it might be useful to document this in other visualizations where you can switch to table view.